.github/maintainers_guide.md

```diff
@@ -3,102 +3,146 @@
-    *  Bump the version number in adherence to [Semantic Versioning](http://semver.org/) in `package.json`.
-    *  Add a changelog entry. Release notes should mention contributors (@-mentions) and issues/PRs (#-mentions). An entry is a new file in `docs/_posts`.
-    *  Commit with a message including the new version number. For example `v1.0.8`.
-    *  Tag the commit with the version number. For example `v1.0.8`.
+**TODO**: verify NPM's policy on revoking a release and version number reuse.
 
-2.  Merge into master repository
-    *  Create a pull request with the commit that was just made. Be certain to include the tag. For
-       example: `git push username master:rel-v1.0.8 && git push --tags username`.
-    *  Once tests pass and a reviewer has approved, merge the pull request. You will also want to
-       update your local `master` branch.
+> If you have any doubt whether your working copy is in good shape to do a release, here is a succinct way to get a
+> fresh start: `npx lerna clean && npx lerna bootstrap`.
 
-3.  Distribute the release
-    *  Publish to the package manager. Once you have permission to publish on npm, you can run `npm publish`.
-    *  Create a GitHub Release. Reuse the release notes written for the changelog entry.
+0. Verify that everything is in order by testing and linting locally before proceeding: `npm run test && npm run lint`.
 
-4.  (Slack Internal) Communicate the release internally. Include a link to the GitHub Release.
+1. Create a tagged commit for the release: `npx lerna version`
+  * Lerna will ask you to make selections for the version increment on each package it plans to tag for release. You
+    should already have an idea of what the appropriate semver increment (patch, minor, or major) you intend to create.
+    If Lerna asks about a package you didn't intend to release, its best to bail at this point
+    (<kbd>CTRL</kbd>+<kbd>C</kbd>).
 
-5.  Announce on Slack Team dev4slack in #slack-api
+2. Publish the release(s) to npm: `NPM_CONFIG_OTP=xxxxxx npx lerna publish from-package`
+  * You should have [2FA set up with NPM](https://docs.npmjs.com/about-two-factor-authentication), and the
+    `NPM_CONFIG_OTP` value should be set to the one time password from your configured second factor device. If the
+    publishing process takes longer than the expiration time of the value (30 seconds), then you may see a publish
+    failure. You can try again as soon as the value changes if you think you can beat the timeout, or you can run
+    `npm publish` in each of the package directories.
 
-6.  (Slack Internal) Tweet? Not necessary for patch updates, might be needed for minor updates,
-    definitely needed for major updates. Include a link to the GitHub Release.
+3. Create GitHub Release(s) and add release notes.
+  * Release notes should mention contributors (@-mentions) and issues/PRs (#-mentions).
+
+4. (Slack Internal) Communicate the release internally. Include a link to the GitHub Release(s).
```

Comment:
Do we want to include (Slack Internal) stuff in the public-facing maintainers guide? Not that it leaks anything, just that it may be confusing to people who aren't Slack staff reading the guide.

Reply:
its been there before, so i guess the precedent has been set. i'm the one who added them before, and i didn't hear anyone push back, but its a decent thing to ask. personally, i think prefixing with `(Slack Internal)` is enough to resolve any potential confusion.